sdk/include/opentelemetry/sdk/common/http_client.h

```diff
@@ -60,18 +60,21 @@ enum class Method
 
 enum class SessionState
 {
-  Created,             // session object is created
-  Ongoing,             // session is ongoing
-  Finished,            // session is finished ( this needs to be the final state )
-  Queued,              // http request is queued
-  TimedOut,            // Request timedout, no response received
-  Aborted,             // http request aborted due to local error,
-  Cancelled,           // http request cancelled, possibly due to session->CancelSession();
-  SendingFailed,       // http request sending failed
+  CreateFailed,        // session create failed
+  Created,             // session created
+  Destroy,             // session destroyed
```

Comment:
Curious - is there a reason to use `Destroy` instead of `Destroyed`?

Reply:
have reverted back to "Destroyed"